libs/application/templates/transport-authority/change-co-owner-of-vehicle/src/forms/ChangeCoOwnerOfVehicleForm/index.ts

```diff
@@ -52,7 +53,7 @@ export const ChangeCoOwnerOfVehicleForm: Form = buildForm({
                   name: payment.general.confirm,
                   type: 'primary',
                   condition: (formValue, externalData) => {
-                    const chargeItemCodes = getChargeItemCodes(
+                    const chargeItemCodes = getChargeItemCodeWithAnswers(
```

Comment:
Because in the Dynamic Condtition check, only formValue and externalData is available. The paymentStateBuilder accepts a function with the application object which is not available within this condition function.